./bin/mutate: line 5: python: command not found
